### DISCUSSION TITLE
base path with name search invalid syntax

### DISCUSSION THREAD

**Discussion by fbmoose48** [vmatare/thinkfan#143] [Support]
when using config:
```
 hwmon /sys/class/hwmon
  name: coretemp
  indices: [1]
```
returns
```
ERROR: /etc/thinkfan.conf:55: Invalid syntax:
name: coretemp
^
```

what is the correct syntax?

Note: the example yaml syntax also seems to be incorrect

**Comment by vmatare** [comment]
What you have posted there is not a valid config file. At the very least, you're missing an enumeration dash in front of `hwmon`. Please read the config manpage: `man thinkfan.conf`. If you still need help after that, you'll have to post your *entire* config (the thing above looks like a snippet to me).
